core/src/test/scala/unit/kafka/server/ShareFetchAcknowledgeRequestTest.scala

```diff
@@ -90,6 +89,7 @@ class ShareFetchAcknowledgeRequestTest(cluster: ClusterInstance) extends GroupCo
     ),
     brokers = 2
   )
+  @Disabled
```

Comment:
Can you write a comment as well regarding why it's diabled. And add a jira for myself to fix this.